image_classification/mobilenet_nchw.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+import {buildConstantByNpy} from '../common/utils.js';
+
+// MobileNet V2 baseline model with nchw layout
+export class MobileNetNchw {
+  constructor() {
+    this.builder_ = null;
+    this.graph_ = null;
+  }
+
+  async buildConv_(input, name, shouldClip = true, options = undefined) {
+    const prefix = './weights/mobilenet_nchw/conv_' + name;
+    const weightsName = prefix + '_weight.npy';
+    const weights =
+        await buildConstantByNpy(this.builder_, weightsName);
+    const biasName = prefix + '_bias.npy';
+    const bias =
+        await buildConstantByNpy(this.builder_, biasName);
+    const conv = this.builder_.add(
+        this.builder_.conv2d(input, weights, options),
+        this.builder_.reshape(bias, [1, -1, 1, 1]));
+    if (shouldClip) {
```

Comment:
please name it `relu6`, that is used by the original model, we just use `clamp` to implement `relu6`.